### PR TITLE
Finnhubb forex provider

### PR DESCRIPTION
**Pull request by monstrobishi**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:

Finnhubb forex provider

#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

File: packages/salmon-provider-finnhubb-forex/__test__/fetch_asset_prices_multiple.test.ts [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    }
+
+    const priceManager = new PriceManager(finnhubbConfig, new FinnhubbForexPriceProvider('API_TOKEN'))
+    const prices = await priceManager.fetchAssetPrices()
+    expect(prices[0].asset).toStrictEqual('GOLD')
+    expect(prices[0].price).toStrictEqual(new BigNumber(1802.532))
+    expect(prices[1].asset).toStrictEqual('EUR')
+    expect(prices[1].price).toStrictEqual(new BigNumber(1.35424))
+    expect(prices[2].asset).toStrictEqual('SGD')
+    expect(prices[2].price).toStrictEqual(new BigNumber(1.0).div(new BigNumber(1.37748)))
```
Comment (jingyi2811) [inline packages/salmon-provider-finnhubb-forex/__test__/fetch_asset_prices_multiple.test.ts:130]: Maybe can change `1.0` to `1`?

File: packages/salmon-provider-finnhubb-forex/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ */
+export class FinnhubbForexPriceProvider implements PriceProvider {
+  constructor (
+    private readonly apiToken: string
+  ) {
+  }
+
+  private async fetchAsset (symbol: string): Promise<AssetPrice> {
+    const interval = parseInt(process.env.INTERVAL_SECONDS ?? '300')
+    const tNow = Math.floor(Date.now() / 1000.0)
```
Comment (jingyi2811) [inline packages/salmon-provider-finnhubb-forex/src/index.ts:24]: Maybe can change `1000.0` to `1000`?

File: packages/salmon-provider-finnhubb-forex/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+    let price = new BigNumber(json.c.slice(-1))
+    if (FINNHUBB_OANDA_SYMBOL_MAPPING[symbol].inverse) {
+      price = new BigNumber(1.0).div(price)
+    }
+
+    return {
+      asset: symbol,
+      price,
+      timestamp: new BigNumber(json.t.slice(-1) * 1000)
```
Comment (jingyi2811) [inline packages/salmon-provider-finnhubb-forex/src/index.ts:44]: Can use bignumber function (Like line 38) to do the multiplication
`new BigNumber(json.t.slice(-1).multipliedBy(1000))`

File: packages/salmon-provider-finnhubb-forex/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    const response = await fetch(fetchPath, {
+      method: 'GET'
+    })
+
+    const text = await response.text()
+    const json = JellyfishJSON.parse(text, 'bignumber')
+
+    let price = new BigNumber(json.c.slice(-1))
+    if (FINNHUBB_OANDA_SYMBOL_MAPPING[symbol].inverse) {
+      price = new BigNumber(1.0).div(price)
```
Comment (jingyi2811) [inline packages/salmon-provider-finnhubb-forex/src/index.ts:38]: Maybe can change `1.0` to `1`?

File: packages/salmon-provider-finnhubb/src/index.ts [hunk 5]
```diff
@@ -26,11 +26,10 @@ export class FinnhubbPriceProvider implements PriceProvider {
     const text = await response.text()
     const json = JellyfishJSON.parse(text, 'bignumber')
 
-    const timestamp = json.t * 1000
     return {
       asset: symbol,
       price: new BigNumber(json.c),
-      timestamp: new BigNumber(timestamp)
+      timestamp: new BigNumber(json.t * 1000)
```
Comment (jingyi2811) [inline packages/salmon-provider-finnhubb/src/index.ts:32]: Can use bignumber function to do the multiplication

**Review by jingyi2811 — APPROVED** [review]

**Review by fuxingloh — APPROVED** [review]

**Pull request merged**